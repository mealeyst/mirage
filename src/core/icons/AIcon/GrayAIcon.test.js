import React from 'react'
import 'jest-styled-components'

import { GrayAIcon } from 'SRC'

const { mountWithTheme } = global

describe('(Styled Component) GrayAIcon', () => {
  const createGrayAIcon = (props) => {
    return mountWithTheme(<GrayAIcon {...props} />)
  }

  test('matching the snapshot', () => {
    expect(createGrayAIcon())
    .toMatchSnapshot()
  })
})
