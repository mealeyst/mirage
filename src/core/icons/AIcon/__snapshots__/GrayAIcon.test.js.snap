// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Styled Component) GrayAIcon matching the snapshot 1`] = `
.c1 {
  width: 9.2rem;
}

.c0 {
  fill: #E6E6E6;
}

<Styled(Styled(AIcon))>
  <Styled(AIcon)
    className="c0"
    width="9.2rem"
  >
    <AIcon
      className="c0 c1"
      width="9.2rem"
    >
      <svg
        className="c0 c1"
        viewBox="0 0 80 80"
        width="9.2rem"
      >
        <path
          d="M57.5,64.6V40.1c0-9.4,0.1-14.8-4.7-19.5c-7.2-7-18.5-7-25.6,0c-4.8,4.8-4.8,10.2-4.8,19.5v24.5h7.4V50.3h20.2 v14.4H57.5z M29.9,43.6V40c0-9.8,0.5-12.5,2.8-15.1c3.9-4,10.5-4,14.5,0l0,0C49.6,27.6,50,30.3,50,40v3.5H29.9z"
        />
      </svg>
    </AIcon>
  </Styled(AIcon)>
</Styled(Styled(AIcon))>
`;
